core/src/software/aws/toolkits/core/region/ToolkitRegionProvider.kt

```diff
@@ -9,7 +9,11 @@ import java.util.concurrent.ConcurrentHashMap
  * An SPI to provide regions supported by this toolkit
  */
 abstract class ToolkitRegionProvider {
-    protected data class PartitionData(val description: String, val services: Map<String, Service>, val regions: Map<String, AwsRegion>)
+    protected data class PartitionData(
```

Comment:
this just cos of new ktlint/detekt rules?

Reply:
yeah i'm planning on doing the detekt changes in a different PR but this didn't get reverted